roles/satellite/tasks/prereq.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: 'Install required packages'
+  package:
+    name: '{{ item }}'
+    state: installed
+  with_items:
+  - satellite
```

Comment:
believe you are looking at pre-6.2 documentation - things have been changed in 6.2 from `katello` to `satellite` - check this out:
https://access.redhat.com/documentation/en-us/red_hat_satellite/6.2/html/installation_guide/installing_satellite_server#installing_satellite_server_install_guide